gto-support-db-coroutines/src/main/kotlin/org/ccci/gto/android/common/db/CoroutinesAsyncDao.kt

```diff
@@ -0,0 +1,24 @@
+package org.ccci.gto.android.common.db
+
+import android.database.sqlite.SQLiteDatabase
+import kotlinx.coroutines.async
+
+interface CoroutinesAsyncDao : CoroutinesDao {
+    fun <T : Any> findAsync(clazz: Class<T>, vararg key: Any) = coroutinesScope.async { find(clazz, *key) }
```

Comment:
Does this [stackoverflow](https://stackoverflow.com/a/2530174) accurately describe the use of the ```clazz``` naming convention?

Reply:
pretty much, `class` is a protected keyword, so you can't use it as a variable name